### PR TITLE
img style fixes, added api movie details

### PR DESCRIPTION
**Pull request by damianmichalowski**
i am added film details from api

### CODE REVIEW

File: src/features/movies/MoviePage/MovieDetails/index.js [hunk 1]
```diff
@@ -20,36 +20,49 @@ import {
...
+export const MovieDetails = (props) => {
+
+    const formatYear = (releaseDate) => {
+        const date = new Date(releaseDate);
+        return date.toLocaleString(
+            undefined, 
+            {
+                year: "numeric",
+            }
+        )
```
Comment (Marcin-Malek) [inline src/features/movies/MoviePage/MovieDetails/index.js:39]: `date.getFullYear();` would do same thing here.

File: src/features/movies/MoviePage/MovieDetails/index.js [hunk 2]
```diff
@@ -20,36 +20,49 @@ import {
...
+                {(props.image && <Image src={props.image} />) || <CameraIcon />}
+            </Poster>
+            <ProductionDetails>
+                <DetailsWrapper>
+                    <Property>Production:</Property>
+                    <Detail>{props.production}</Detail>
+                </DetailsWrapper>
+                <DetailsWrapper>
+                    <Property>Release date:</Property>
+                    <Detail>{props.release}</Detail>
```
Comment (Marcin-Malek) [inline src/features/movies/MoviePage/MovieDetails/index.js]: This date has to be parsed too.

File: src/features/movies/MoviePage/index.js [hunk 3]
```diff
@@ -1,30 +1,54 @@
...
+                        image={`https://image.tmdb.org/t/p/w1280/${movieDetails.backdrop_path}`}
+                        rating={movieDetails.vote_average}
+                        votes={movieDetails.vote_count} />
+                    <MainWrapper>
+                        <MovieDetails
+                            image={`https://image.tmdb.org/t/p/w500/${movieDetails.poster_path}`}
+                            title={movieDetails.original_title}
+                            year={movieDetails.release_date}
+                            production={movieDetails.production_countries[0].name}
+                            release={movieDetails.release_date}
```
Comment (Marcin-Malek) [inline src/features/movies/MoviePage/index.js]: You dont have to pass release_date twice. You could do `date={movieDetails.release_date}` like in MovieTile

File: src/features/movies/MoviePage/index.js [hunk 4]
```diff
@@ -1,30 +1,54 @@
...
+                        title={movieDetails.original_title}
+                        image={`https://image.tmdb.org/t/p/w1280/${movieDetails.backdrop_path}`}
+                        rating={movieDetails.vote_average}
+                        votes={movieDetails.vote_count} />
+                    <MainWrapper>
+                        <MovieDetails
+                            image={`https://image.tmdb.org/t/p/w500/${movieDetails.poster_path}`}
+                            title={movieDetails.original_title}
+                            year={movieDetails.release_date}
+                            production={movieDetails.production_countries[0].name}
```
Comment (Marcin-Malek) [inline src/features/movies/MoviePage/index.js]: I just found out that in our UI project all of production countries are presented, so we should pass all production countries and then map them in MovieDetails component

File: src/features/movies/moviesSlice.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
     moviesList: [],
-    moviesGenres: [],
+    movieDetails: [],
```
Comment (Marcin-Malek) [inline src/features/movies/moviesSlice.js]: we need moviesGenres here. And movieDetails should be an empty object rather than empty array.

**Review by Marcin-Malek — COMMENTED** [review]
Great job! 
I could not comment it but you took back redux logo in public folder we got rid of in a different branch - we dont need them anymore.

File: src/features/movies/MoviePage/MovieDetails/index.js (hunk 1 above)
Comment (damianmichalowski) [reply src/features/movies/MoviePage/MovieDetails/index.js:39]: right

File: src/features/movies/MoviePage/MovieDetails/index.js (hunk 2 above)
Comment (damianmichalowski) [reply src/features/movies/MoviePage/MovieDetails/index.js]: i resolve this adding new functoin 
`const formatDate = (releaseDate) => {
        const date = new Date(releaseDate);
        return date.toLocaleString(
            undefined, 
            {
                day: "numeric",
                month: "numeric",
                year: "numeric"
            }
        )
    };`

File: src/features/movies/MoviePage/index.js (hunk 3 above)
Comment (damianmichalowski) [reply src/features/movies/MoviePage/index.js]: done

File: src/features/movies/MoviePage/index.js (hunk 4 above)
Comment (damianmichalowski) [reply src/features/movies/MoviePage/index.js]: done

File: src/features/movies/moviesSlice.js (hunk 5 above)
Comment (damianmichalowski) [reply src/features/movies/moviesSlice.js]: fix

**Review by JamieShifter — APPROVED** [review]
good job

**Review by Marcin-Malek — APPROVED** [review]
Great Job! Keep it up.

**Pull request merged**